src/feature-test/java/uk/gov/ida/verifyserviceprovider/configuration/VerifyHubConfigurationFeatureTests.java

```diff
@@ -79,26 +92,20 @@ public void shouldSetExpectedDefaultsForProductionEnvironmentIfNoOverridesGiven(
 
     @Test
     public void shouldSetHubMetadataExpectedEntityIdToTheConfigValueIfOneHasBeenProvided() throws Exception {
-        String config = "{" +
-            "\"environment\": \"INTEGRATION\"," +
-            "\"metadata\": {" +
-            "\"expectedEntityId\": \"some-expected-entity-id\"" +
-            "}}";
+        String config = "{\"environment\": \"CUSTOM\"}";
+        setUpEnvironmentForCustom();
 
         VerifyHubConfiguration actualConfiguration = OBJECT_MAPPER.readValue(config, VerifyHubConfiguration.class);
 
-        assertThat(actualConfiguration.getHubMetadataConfiguration().getExpectedEntityId()).isEqualTo("some-expected-entity-id");
+        assertThat(actualConfiguration.getHubMetadataConfiguration().getExpectedEntityId()).isEqualTo("http://some-expected-entity-id");
     }
 
     @Test
     public void shouldSetHubMetadataTrustStorePathToTheConfigValueIfOneHasBeenProvided() throws Exception {
-        String config = "{" +
-            "\"environment\": \"PRODUCTION\"," +
-            "\"metadata\": {" +
-            "\"trustStore\": {" +
-            "\"path\": \"" + keyStore.getAbsolutePath() + "\"," +
-            "\"password\": \"" + keyStore.getPassword() + "\"" +
-            "}}}";
+        String config = "{\"environment\": \"CUSTOM\"}";
+        setUpEnvironmentForCustom();
+        environmentVariables.set(METADATA_TRUSTSTORE_PATH, keyStore.getAbsolutePath());
+        environmentVariables.set(TRUSTSTORE_PASSWORD, keyStore.getPassword());
```

Comment:
setUpEnvironmentForCustom() already covers line 107 and 108. Can these lines (107 & 108) be removed?